### DISCUSSION TITLE
Possible issue with multiple instances of Chrome

### DISCUSSION THREAD

**Discussion by optimusdag** [will-stone/browserosaurus#605] [Q&A] [answered]
Hi.
I have a main instance of Chrome that I use for most purposes and, for development purposes, I use a second instance opened like this:
`open -n /Applications/Google\ Chrome.app --args --disable-web-security --user-data-dir=/Users/blah/blah/blah/dev-profile`

The thing is that since I run this secondary instance, Browserosaurus is only opening links on this "development" instance and does not let me use my main one. 

Is there a way to force the use of the main instance? Or at least for it to use the last executed instance?

**Comment by will-stone** [comment, marked as answer]
Hmmm, not sure. Here's the code for opening an app if you'd like to have a play: https://github.com/will-stone/browserosaurus/blob/main/src/main/utils/open-app.ts#L17

**Reply by optimusdag** [reply]
I see. Is there a way to manually change the list of browsers and maybe add optional command line parameters?

**Reply by will-stone** [reply]
Not currently, not without forking the project. I take the view of keeping B as easy for the average user (me) as possible. However, I am toying with the idea of moving the preferences into a `~/.browserosaurus.js` file. But currently feel this is going to cause much more noise and pain to the B community than I'd be willing to take 😭

**Reply by optimusdag** [reply]
Ok. If I have some time, I'll take a look to see if I can make a non-breaking change to allow that and maybe submit a PR. Thanks

**Reply by optimusdag** [reply]
It was fun to mess around with the code and I found a way to add/remove browsers and add optional command line parameters on the store.json. Unfortunately, I did not manage to get the desired results from the command line itself, since Chrome refuses to select an already open profile. :(
